src/Compilers/Core/Portable/Emit/CommonPEModuleBuilder.cs

```diff
@@ -41,6 +40,8 @@ internal abstract class CommonPEModuleBuilder : Cci.IUnit, Cci.IModuleReference
         // Only set when running tests to allow realized IL for a given method to be looked up by method.
         internal ConcurrentDictionary<IMethodSymbol, CompilationTestData.MethodData> TestData { get; private set; }
 
+        internal readonly DebugInformationFormat DebugInformationFormat;
```

Comment:
I found this field hard to remove, but I'm open to push more on that. One of the usages (in `EmitStateMachineScope` in VB I think) was particularly difficult to thread through.